src/rgw/rgw_reshard.cc

```diff
@@ -993,8 +989,11 @@ int RGWReshard::process_single_logshard(int logshard_num)
 	}
 
 	RGWBucketReshard br(store, bucket_info, attrs, nullptr);
-	ret = br.execute(entry.new_num_shards, max_entries, false, nullptr,
-			 nullptr, this);
+        // ret = br.execute(entry.new_num_shards, max_entries, false, nullptr,
+	// 		 nullptr, this);
+        Formatter* formatter = new JSONFormatter(false);
```

Comment:
i think this is a leak. you should probably use unique_ptr

Reply:
Removed & apology, this hunk was committed by mistake,
(it was changed for testing purpose only).
I did not intend to revert commit cc27e353cb821dcba6f1a484bf2ea65a0c109778